qiita_pet/handlers/websocket_handlers.py

```diff
@@ -74,7 +74,24 @@ def on_close(self):
         self.redis.disconnect()
 
 
+class SelectedSocketHandler(WebSocketHandler, BaseHandler):
+    """Websocket for removing samples on default analysis display page"""
+    @authenticated
+    def on_message(self, msg):
+        # When the websocket receives a message from the javascript client,
+        # parse into JSON
+        msginfo = loads(msg)
+        default = Analysis(self.current_user.default_analysis)
+        if 'samples' not in msginfo:
```

Comment:
I think you can simplify this if/else block by using msginfo.get('samples'), which would return a `None` when `'samples'` is not part of the dictionary. And perhaps, this is the way both arguments should be retrieved such that in case nothing was sent remove_samples would raise an exception (QiitaIncompetentDeveloper, I think).